### PR TITLE
fix(graph): manage timezone to display graphs

### LINKED ISSUE #6022: Timezone problem in graphs
---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------

**Centreon Web version**: 2.8.15

**Centreon Engine version**: 1.8.1

**Centreon Broker version**: 3.0.11

**OS**: RH 6.9

**Steps to reproduce the issue:**
1. Go to : Monitoring > Performance
2. The timezone does not correspond to that of the user

**Describe the results you received:**

image

**Describe the results you expected:**

After some modification: 

image

### PR DESCRIPTION
**Pull request by kduret**
<h2> Description </h2>

Fixes #6022

<h2> Type of change </h2>

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

<h2> Target serie </h2>

- [ ] 2.8.x
- [x] 18.10.x
- [x] 19.04.x (master)

### CODE REVIEW

File: www/include/views/graphs/javascript/centreon-graph.js [hunk 1]
```diff
@@ -542,20 +548,39 @@
...
-          [":%S", function(d) { return d.getSeconds(); }],
-          ["%H:%M", function(d) { return d.getMinutes(); }],
-          ["%H:%M", function(d) { return d.getHours(); }],
-          ["%m-%d", function(d) { return d.getDay() && d.getDate() !== 1; }],
-          ["%m-%d", function(d) { return d.getDate() !== 1; }],
-          ["%Y-%m", function(d) { return d.getMonth(); }],
-          ["%Y", function() { return true; }]
-        ]);
+          timeFormat = function(date) {
+              // convert to moment object to mange timezone
```
Comment (sc979) [inline www/include/views/graphs/javascript/centreon-graph.js]:
```suggestion
              // convert to moment object to manage timezone
```

File: www/include/views/graphs/javascript/centreon-graph.js [hunk 2]
```diff
@@ -30,6 +30,12 @@
       });
     }
 
+    // get timezone from localstorage
+    // be careful, it seems that it's updated when user logout/login
+    this.timezone = localStorage.getItem('realTimezone')
+        ? localStorage.getItem('realTimezone')
```
Comment (sc979) [inline www/include/views/graphs/javascript/centreon-graph.js]: this js file is indented using two spaces

**Comment by CPbN** [comment]
While in graphs, perhaps these other graph related PR could be reviewed / added  to the [19.04.1 project](https://github.com/centreon/centreon/projects/39) ?
7343, 7341, 7316, 7314
Many thanks :)

**Comment by kduret** [comment]
@lpinsivy could you prioritize those PRs please ? :)

**Pull request merged**